### ISSUE TITLE
Categories don't appear to be used on any page?

### ISSUE THREAD

**Issue by craigmaloney** [GetTogetherComm/GetTogether#192]
Noticed that while I can set a category it doesn't appear those categories are used on the site. I can't filter on categories, nor can I see what a particular group considers itself.

**Comment by mhall119** [comment]
Currently they're only used to provide a default image for a team and it's events, each category has their own. I had planned to use it to filter by category, send recommendations to users when a new team or event was created near them, etc. It hasn't been a pressing need since gettogether.community doesn't yet have that many teams that filtering on anything but location is needed.

**Comment by craigmaloney** [comment]
Right, but I wanted to see if I could get some more of the local groups together to get onto the site so I can get out of the "maintaining links to other groups" business. :grin:

**Comment by mhall119** [comment]
It would be relatively easy to add it to the homepage search functionality, although that search bar is getting rather crowded now and might need a redesign soon.

**Comment by mhall119** [comment]
I'm limiting the scope of this issue to just being able to filter by category, leaving the other category-based features I planned to other issues

**Comment by dm20** [comment]
Hey @mhall119 @craigmaloney - I'm a software engineer at Indeed.com looking for Hacktoberfest issues and this caught my eye!
Is this sort of what you have in mind? Would the category input also be a drop down list? 

<img width="1363" alt="Screen Shot 2019-10-17 at 5 28 56 PM" src="https://user-images.githubusercontent.com/16715394/67057084-ed4db700-f103-11e9-92fb-c65469793567.png">


I'd be happy to take this on, following your existing design patterns of course!

**Comment by mhall119** [comment]
Hi Daniel,
I think the search bar is getting a little too crowded, especially when viewed on a mobile device. I'm not sure there's a real benefit to having a team name search there anymore, so maybe we can remove that to make room.
I think I would prefer the City field to be the first one, as likely to be the most frequently used. Thoughts on that?

**Comment by craigmaloney** [comment]
Perhaps moving the team search over to "Teams / All Teams" might help, especially if someone is looking for a specific team (aka: Ubuntu groups will likely have "Ubuntu" in the name).

**Comment by mhall119** [comment]
That's a good idea.
